docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+real Conjur server, and it uses standard secure technology to encrypt your
+secrets in transit. This means:
+
+* Your Conjur server will be `https:` instead of `http:`, just like a secure
+  website
+* Because the client knows it's talking to the real server, the "man in the
+  middle" will be exposed and the client won't leak any secret information
+* Since the traffic to and from Conjur is scrambled using secure encryption,
+  passive listeners on your network can't learn anything about the contents of
+  your secrets
+
+For these reasons, it is crucial to set up TLS correctly when you deploy Conjur.
+
+### Protect Your Secrets
+
+Do not leave this to chance: even a small flaw can totally compromise your
+secrets. Setting up Conjur and TLS without the appropriate expertise is like
+packing your own parachute and jumping out of a plane.
+
+That doesn't mean you should close the tab and walk away. It means you should get
+in touch with us and your own security team so we can ensure that you can employ
+Conjur successfully.
+
+[authn]: https://www.conjur.org/api.html#authentication-authenticate-post
+[secret-get]: https://www.conjur.org/api.html#secrets-retrieve-a-secret-get
+
+## Using NGINX To Proxy Traffic With TLS To Conjur
+
+[NGINX][NGINX] is a web proxy that's Free Software and easy to configure. This
+tutorial will show you how to use Docker to install Conjur and NGINX and
+configure them to provide Conjur with TLS.
+
+[NGINX]: https://www.nginx.com
+
+### Prerequisites
+
+This tutorial requires Docker and a terminal application. Prepare by following
+the prerequisite instructions found on [Install Conjur][prerequisites].
+
+[prerequisites]: /get-started/install-conjur.html#prerequisites
+
+Additionally, you will need the tutorial files from the Conjur source code
+repository. Here's how you get them:
+
+1. Install Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+1. Clone the Conjur repository
+   
+   In your terminal application, run:
+   ```sh-session
+   $ git clone https://github.com/cyberark/conjur.git
+   ```
+   This will create a folder called `conjur` in the current directory.
+
+### The Good Part
+
+To start out our experience on a high note, let's get the full Conjur+TLS stack
+up and running so we can inspect it.
+
+The tutorial script will install Conjur and NGINX, configure them to work
+together, and connect a Conjur client to the running server. This is a full
+end-to-end working installation to allow you to see how the pieces fit.
+
+```sh-session
+$ # start the Conjur+NGINX tutorial servers
+$ cd conjur/docs/tutorials/integrations/nginx
+$ ./start.sh
+```
+
+Take a look at the logs for the Conjur or NGINX servers:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
+$ docker-compose logs conjur
+$ docker-compose logs proxy
+```
+
+Run some commands in the Conjur client:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
```

Comment:
the way you've rewritten this, I think this line isn't needed anymore

Reply:
ok